### PR TITLE
feat: added context

### PR DESCRIPTION
**Pull request by Avin008**
### Context Includes:

- Filter-Context
- Notes-Context

### CODE REVIEW

File: src/components/CreateNote/create-note.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  border: 1px solid var(--color-black);
+  width: 37.5rem;
+  padding: 0.5rem;
+  background-color: var(--note-bg-color);
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  position: absolute;
+  border-radius: 0.5rem;
+  box-shadow: rgba(28, 28, 33, 0.2) 0px 7px 29px 0px;
```
Comment (TusharJade) [inline src/components/CreateNote/create-note.css:18]: you can convert it into hex color

**Review by TusharJade — APPROVED** [review]
Looks good , code is clean good to go for merge ✅

File: src/components/CreateNote/CreateNote.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      </div>
+      <div className="note-footer">
+        <select
+          name="label"
+          id="select-label"
+          onChange={(e) =>
+            setNotesData((prev) => ({ ...prev, label: e.target.value }))
+          }
+        >
+          <option value="">choose label</option>
```
Comment (Kothalkarpriya) [inline src/components/CreateNote/CreateNote.jsx:75]: the value should not be empty I think. If not needed you can remove or you have forgotten to place value into it

**Review by Kothalkarpriya — APPROVED** [review]
Keep PR small for Reviews, otherwise Good work

**Review by faisal-kursheedali — COMMENTED** [review]
looks good to me, good to go.

**Pull request merged**